### DISCUSSION TITLE
Behaviour of --once with period option

### DISCUSSION THREAD

**Discussion by lunik1** [troglobit/inadyn#447] [Q&A]
I have a question on how running `inadyn` in `--once` mode interacts with the `period` option. Am I correct that this setting is ignored when running in oneshot mode? Or will oneshot mode do nothing (not even check the ip) if the length of time between invocations is less than `period`? Or something else?

**Comment by troglobit** [comment]
Yes, setting 'once' overrides the number of iterations to '1' internally.
